integrations/unity-csharp/Assets/VDK/VaultAPI.cs

```diff
@@ -0,0 +1,544 @@
+using System;
```

Comment:
This entire file is weird; There are weird, half finished implementation details everywhere- Things like missing `[StructLayout(LayoutKind.Sequential)]`

How many of these changes are required and how much was tested?

Reply:
Fixed the structs, API changes should be correct now